pkg/middlewares/iam_policy_attachment_expander_test.go

```diff
@@ -287,7 +287,7 @@ func TestIamPolicyAttachmentSanitizer_Execute(t *testing.T) {
 						Type: aws.AwsIamPolicyAttachmentResourceType,
 						Attrs: &resource.Attributes{
 							"policy_arn": "fromstatearn",
-							"roles":      []string{"role1"},
+							"roles":      []interface{}{"role1"},
```

Comment:
What type can this field be other than []string ?

Reply:
None I guess